src/components/molecules/card/card.scss

```
@@ -0,0 +1,65 @@
+@use "../../variables" as *;
+
+.card {
+  padding: 0;
+  background-color: $white_smoke;
+
+  .card-img-all {
+    position: relative;
+    overflow: hidden;
+    .card-img {
+      transition: transform 0.5s ease;
+    }
+
+    &:hover .card-img {
+      transform: scale(1.2);
+    }

```

現在imgの拡大を確認できていない状態です。
おそらく必要のないものを記述しているからだと思うので以下に修正して確認お願いします！
```suggestion
    &:hover {
      transform: scale(1.2);
    }

```